### DISCUSSION TITLE
Can i get r, s, z, Pubkey from this tutorial?

### DISCUSSION THREAD

**Discussion by Ango87** [Congyuwang/Rusty-Bitcoin-Explorer#12] [General]
Can I get the value of R, s, s1, z, z1, Pubkey through this tutorial, when i have two transactions from bitcoin address !??

**Comment by Congyuwang** [comment]
I don’t understand you question. Explain R s s1 z z1.

**Comment by Ango87** [comment]
@Congyuwang These are the values that we can get out with Bitcoin transactions

**Reply by Congyuwang** [reply]
Use ‘Block’ in stead of ‘FBlock’ or ‘SBlock’ so you can get raw transaction

**Reply by Congyuwang** [reply]
Interesting though, what do you plan to do with these information

**Comment by Congyuwang** [comment]
Sorry. Currently this crate does not support extraction of R, S, Z. Though, you might consider contribute to this repo!

**Reply by Ango87** [reply]
😭😭😭😭😭
